packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/cultureAssessments.js

```diff
@@ -37,7 +37,7 @@ const cultureAssessments = [
 		nonInclusivePhrases: [ "tribe" ],
 		inclusiveAlternatives: "<i>group, cohort, crew, league, guild</i>",
 		score: SCORES.POTENTIALLY_NON_INCLUSIVE,
-		feedbackFormat: potentiallyHarmfulUnlessCulture,
+		feedbackFormat: potentiallyHarmfulUnlessCulture.slice( 0, -42 ) + "a culture that uses this term.",
```

Comment:
It is quite opaque what the slicing operation is meant to achieve. It takes quite some effort to track down exactly what part is ignored. For readability it would help to add a comment explaining the goal of the slicing operation.

Even more ideal would be another variable with the other text. But I think that is too much refactoring for this stage. But we could keep this into mind for the future.